### ISSUE TITLE
So many questions i cant find answers for

### ISSUE THREAD

**Issue by TeKett** [bmaltais/kohya_ss#1839]
I asked over on the discussion thread, but got no answers.
The current ones on my mind is:

What is the actual resulting difference quality wise between using different batch sizes? Is 1 better then 2 when using very varied image set?

What is the difference between Learning Rate, Text Encoder learning rate, and Unet learning rate? How do they function together? When i don't have the same on all 3 i get wonky images. If im training style its either not consistent if Text Encoder is higher or doesn't pick it up at all if Unet is lower.

How does the LR scheduler affect things? Since besides the initial setting on the 3 mentioned above, it dosn't seem to do much if anything for the result. And any training beyond 1-3 epochs don't seem to do anything.

Should i or shouldn't i use a VAE? Preview images have better quality, but would this affect using a VAE when generating images later? Theres no cost that i can see.

Number of cpu threads per core? Should this just match what i have? 2 with hyperthreading and 1 without? What happens if i set it to more then what i have? How does this affect the new consumer intel cards that only has hyperthreading on the first 8 and not the rest of the cores. I know that the cpu dont play a huge roll as compared to the gpu, so it might not even matter.

How does max token length work? If i use 75 on a caption 150 long does it discard the second half? Does it go before or after `--caption_tag_dropout_rate`? Why is there a limit?

Can i train at multiple resolutions even if im only going to be generating images at a single one, or is this wasted training? As in should i make all training images the exact same size or not, or at least one of the directions the same, AKA should i use no/1, or 10000 buckets? Would it be beneficial to train at 2x size? Since in the end im going to use 2x when generating images.

When looking at bitsandbytes files i noticed it has files for cuda122, looking som more and cuda 123 was released September. Why hasn't the SD community upgraded from 118, considering 119 was released all the way back in april. Is there no benefit from using these versions? There's 0 related results when searching about it.
